cleo_plugins/DebugUtils/DebugUtils.cpp

```diff
@@ -183,7 +183,7 @@ class DebugUtils
 
     static bool WINAPI OnScriptProcess(CScriptThread* thread)
     {
-        currScript.Reset();
```

Comment:
any idea why `Reset` did not work for plg-sdk script called from ASI?

Reply:
Plugin SDK just calls ProcessOneCommand.
https://github.com/DK22Pac/plugin-sdk/blob/master/shared/extensions/ScriptCommands.h#L147

Script is not added to any queue and OnScriptProcess is not called either.

Reply:
But calling reset from ongameprocess hook didn't seem to help either. Weird. Is ongameprocess called each frame?

Reply:
All scripts were processed, last script's command count was not cleared and was above the limit due to *.cs4 mode.
Then during same frame ASI plugin started calling it's commands. So every single fake script from SDK was throwing error.

Reply:
Aaah, so we counted cs4 instructions towards the limit, but displayed the error for a non-cs4 script. Makes sense now